### ISSUE TITLE
chart.update does not update chart.accessibility.pointDescriptionFormatter

### ISSUE THREAD

**Issue by ejubkadric94** [highcharts/highcharts#9669]
Function `update` successfully changes any chart option that I need to change, except `accessibility.pointDescriptionFormatter`. I tried, but it does not work. Please see the attached fiddle

#### Expected and Actual behaviour
In this fiddle I created a basic column chart with one point. 
I defined `tooltip.pointFormatter` and `accessibility.pointDescriptionFormatter`. So if you hover over the point, you will see a tooltip with the value defined in tooltip point formatter. 
Also, if you open chrome inspector, and inspect the point itself, you will see that it has `aria-label="FIRST aria-label description"` attribute, as it is defined in accessibility point descrtiption formatter.

Then, you will click the button below the chart, and a function will be triggered. That function updates the chart with new `tooltip.pointFormatter` and `accessibility.pointDescriptionFormatter` callbacks, which return different values.
You will notice that the point tooltip is indeed changed, but the `aria-label` on point is not changed.
https://jsfiddle.net/5kn9h8oy/18/

#### Product version
Highcharts 6.2.0
Tested on latest highcharts as well, where bug is also present.

#### Affected browser(s)
Google Chrome Version 70.0.3538.110 (Official Build) (64-bit)

**Comment by sebastianbochan** [comment]
Hi @ejubkadric94,
Thank you for reporting.

**Internal note:**
It looks like these lines 
```
            if (chart[key] && typeof chart[key].update === 'function') {
                chart[key].update(val, false);
            // If a one-to-one object does not exist, look for an adder function
            } else if (typeof chart[adders[key]] === 'function') {
                chart[adders[key]](val);
            }
```

in the `chart.update()` ignore the accesibility.

The solution could be add `update()` method to accesibility object. 

What do you think about this @oysteinmoseng ?
